examples/cosmos/sdk_boundary_test.go

```diff
@@ -37,9 +37,7 @@ func TestSDKBoundaries(t *testing.T) {
 				},
 				{
 					Name: "ibc-go-simd", ChainName: "simd-50", Version: "feat-upgrade-sdk-v0.50", //sdk 0.50 alpha
-					ChainConfig: ibc.ChainConfig{
-						UsingNewGenesisCommand: true,
-					},
+					ChainConfig: ibc.ChainConfig{},
```

Comment:
The `ChainConfig` can be removed entirely now since there are no overrides

Reply:
Resolved: def97dfbd2b16619714919dfa9b0b0a17ee071bb